tests/countries/test_india.py

```
@@ -917,3 +917,15 @@ def test_deprecated(self):
         self.assertEqual(
             India(subdiv="OR", years=2023).keys(), India(subdiv="OD", years=2023).keys()
         )
+
+    def test_holi_maharashtra_2026(self):
+        holidays = India(years=2026, subdiv="MH")
+
+        # correct date present
+        self.assertIn("2026-03-03", holidays)
```

Ruff (0.15.6)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)